adagrams/game.py

```diff
@@ -1,11 +1,149 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+
+    aux_letter_pool = LETTER_POOL.copy()
+    
+    available_letters_list = []
+
+    while len(available_letters_list) < 10:
+        numbers = (random.randint(65,90))
+        letters_for_user = chr(numbers)
```

Comment:
Technically, I think that this solution doesn't take into account the frequency of each letter. For example, I'm pretty sure each letter has an equal 1/26th of being drawn. This is a minor bug that we don't test for, but it's worth considering.